task/sp_measure/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "task-sp-measure"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+userlib = {path = "../../sys/userlib", features = ["panic-messages"]}
+ringbuf = {path = "../../lib/ringbuf"}
+drv-sp-ctrl-api = {path = "../../drv/sp-ctrl-api"}
+sha2 = {version = "0.9", default-features = false}
+
+[build-dependencies]
+build-util = {path = "../../build/util"}
+idol = {git = "https://github.com/oxidecomputer/idolatry.git"}
+anyhow = "1.0.31"
+serde = "1"
+quote = "1"
+sha2 = {version = "0.9", default-features = false}
```

Comment:
We're currently using `sha3-256` in sprockets and in the early-boot dice stuff that @flihp is doing. We should probably use that here also.

Reply:
I've just updated RFD36 to reflect this. I was using sha512 there to do "worst-case" storage analysis but hadn't updated it to include the determination we made recently w/r/t sha3-256: https://github.com/oxidecomputer/rfd/commit/a19b0e7187048afeab44f05b2bb55154a5542b24